notifications-backend/pom.xml

```
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>notifications-backend</artifactId>
+
+    <parent>
+        <groupId>com.redhat.cloud.notifications</groupId>
+        <artifactId>notifications</artifactId>
```

Shouldn't this be
```suggestion
        <artifactId>notifications-backend</artifactId>
```
?

---

I would say no as we want the root project to be renamed from notifications-backend to just notifications. So the parent should be notifications instead of notifications-backend as well.

---

I edited my previous comment too late. Everything's fine, my comment was wrong.